### PR TITLE
Testimonial list

### PR DESCRIPTION
**Pull request by sarunast**
I added Testimonial List on `aktive Werden` page. The page itself will be changed this week so I just added for demonstration.

screen shot 2018-02-13 at 12 05 43
screen shot 2018-02-13 at 12 06 06
screen shot 2018-02-13 at 12 06 23

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 3fc0673ed10d5d840e12d96763dbe11a7ed533c2

https://deploy-preview-162--masifunde.netlify.com

File: components/TestimonialList/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    margin-bottom: ${extraSmallSpacing};
+    text-align: center;
+  }
+`
+
+const PatronList = ({ data: patrons, className }) => (
```
Comment (denisw) [inline components/TestimonialList/index.js]: Shouldn't that be `TestimonialList`?

File: components/TestimonialList/Patron.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  margin-bottom: 0.5rem;
+`
+
+const Title = styled.div`
+  font-size: ${rem('12px')};
+  line-height: 1.43;
+  color: ${({ theme }) => theme.pineCone};
+`
+
+const Patron = ({
```
Comment (denisw) [inline components/TestimonialList/Patron.js]: Shouldn't that be `Testimonial`?

File: styling/utils.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { css } from 'styled-components'
+
+// eslint-disable-next-line import/prefer-default-export
+export const wordBreak = css`
+  // Add line breaks 
+  // Source: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/
+
+  /* These are technically the same, but use both */
```
Comment (denisw) [inline styling/utils.js]: Why do we use both?

File: styling/utils.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+
+  -ms-word-break: break-all;
+  /* This is the dangerous one in WebKit, as it breaks things wherever */
+  word-break: break-all;
+  /* Instead use this non-standard one: */
+  word-break: break-word;
+
+  /* Adds a hyphen where the word breaks, if supported (No Blink) */
```
Comment (denisw) [inline styling/utils.js]: Great documentation work! 👍

File: components/TestimonialList/index.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+          </div>
+        </div>
+      </PatronContainer>
+    ))}
+  </div>
+)
+
+PatronList.propTypes = {
+  className: PropTypes.string,
+  data: PropTypes.arrayOf(PropTypes.shape({
```
Comment (denisw) [inline components/TestimonialList/index.js]: Could this prop have a more descriptive name? `testimonials` perhaps?

File: components/TestimonialList/index.js (hunk 1 above)
Comment (sarunast) [reply components/TestimonialList/index.js]: Yeah, you are right. In the beginning, I named it as a `PatronList` and then renamed the file, but forgot to do with the actual names inside the file.

File: styling/utils.js (hunk 3 above)
Comment (sarunast) [reply styling/utils.js]: This is the explanation from the website:

Comment (denisw) [reply styling/utils.js]: I see. How about simply extending the comment to read `..., but use both for broader browser support`? That would bring the point across without having to read the article.

File: components/TestimonialList/index.js (hunk 5 above)
Comment (sarunast) [reply components/TestimonialList/index.js]: I had it in the begining as you suggested but than I was looking in

Comment (sarunast) [reply components/TestimonialList/index.js]: I had it in the beginning as you suggested. I was looking in two options `<TestimonialList testimonials={testimonials} />` vs `<TestimonialList data={testimonials} />` . The component name says `TestimonialList` that's why I switched to `data`. I can change it if you think that is better. I don't have strong feelings about either of these 😄

Comment (denisw) [reply components/TestimonialList/index.js]: I think a `TestimonialList` having a prop `testimonials` is the most natural thing in the world. 😉 Don't let the seeming name duplication scare you!

File: components/TestimonialList/Patron.js (hunk 2 above)
Comment (sarunast) [reply components/TestimonialList/Patron.js]: Now that I think more it is only the Image of the person with `name` and `title`.  I doesn't have anything to do with testimonial itself.
screen shot 2018-02-13 at 17 33 40

Comment (sarunast) [reply components/TestimonialList/Patron.js]: As discussed I will keep this name `Patron`

**Comment by sarunast** [comment]
Checked out with IE11 there was a gap between image and text on a small resolution. Fixed that.

**Pull request merged**